build/dependencies.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <AspNetCoreIntegrationTestingVersion>0.5.0-*</AspNetCoreIntegrationTestingVersion>
-    <AspNetCoreVersion>2.1.0-*</AspNetCoreVersion>
+    <AspNetCoreIntegrationTestingVersion>0.4.0-*</AspNetCoreIntegrationTestingVersion>
```

Comment:
I'd recommend removing floating versions altogether. We're ditching them at the moment with "lineups", but since SignalR will be out-of-band, we won't have a lineup for you to use. You'll reduce build instability by hard coding versions.